app/frontend/src/application/jobseekers/map.js

```diff
@@ -14,30 +14,43 @@ const getSchool = () => {
   return false;
 };
 
+// For search result map
+const getPolygonBoundaries = () => {
```

Comment:
Not sure how I feel about this pattern but I can see it's the same as already exists in this file - maybe one to refactor a bit later.